sdk/core/core-http/lib/policies/logPolicy.ts

```diff
@@ -36,12 +36,12 @@ export interface LoggingOptions {
   /**
    * The Debugger (logger) instance to use for writing pipeline logs.
    */
-  logger?: Debugger,
+  logger?: Debugger;
 
   /**
    * Options to pass to the logPolicy factory.
    */
-  logPolicyOptions?: LogPolicyOptions
+  logPolicyOptions?: LogPolicyOptions;
```

Comment:
Now that this isn't the one being passed into LogPolicy directly, it'd be nice to have a more appropriate name (or lift the fields into LoggerOptions?).  Not sure it's worth going to ship room over, though...